### PR TITLE
Adds ability to save via keyboard by pressing ctrl+s

### LINKED ISSUE #29: Support CTRL+S key binding to save challenge
Press `CTRL+S` key to save challenge.

### PR DESCRIPTION
**Pull request by terakilobyte**
Closes #29.

### CODE REVIEW

**Comment by SaintPeter** [comment]
While this does work - it saves the file - it also prompts me to save the .html file.  On Chrome on Windows. 

image

I think you need to prevent the default action from occurring:
http://stackoverflow.com/questions/4446987/overriding-controls-save-functionality-in-browser

File: src/Components/GrandCentralStation.js [hunk 1]
```diff
@@ -68,6 +69,13 @@ class GrandCentralStation extends Component {
     });
   }
 
+  handleKeyboardSave(e) {
+    if (e.keyCode === 83 && e.ctrlKey) {
+      this.modalSave();
+    }
```
Comment (SaintPeter) [inline src/Components/GrandCentralStation.js:76]: add `e.preventDefault()`?

**Comment by terakilobyte** [comment]
It didn't occur to me at first to swallow the event as I use a sensible OS ;)

I've added `e.preventDefault()` to swallow the event and not trigger default OS dialogues.

**Comment by SaintPeter** [comment]
Works great now.  Thanks!

**Pull request merged**